src/components/play-page.js

```diff
@@ -165,9 +166,11 @@ export class FlashCards extends connect(store)(PageViewElement) {
         return;
       }
       const activeCardData = this._cards[activeCard.hint][activeCard.index];
+      var answers = [].concat.apply([], [activeCardData.en]);
       this._card = {
         question: activeCardData.jp,
-        answer: activeCardData.en,
+        _answers: answers,
+        answer: answers[0],
```

Comment:
only one of these is needed, and should be named as a public variable either way :)